miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java

```diff
@@ -210,6 +214,37 @@ public ModelAndView setupForm(@PathVariable Long poolId, ModelMap model) throws
     }
   }
 
+  private void addWarnings(Pool pool, ModelMap model) throws JsonProcessingException {
+    ObjectMapper mapper = new ObjectMapper();
+    Set<String> duplicates = pool.getDuplicateIndicesSequences();
+    Set<String> nearDuplicates = pool.getNearDuplicateIndicesSequences();
+    model.put("duplicateIndicesSequences", mapper.writeValueAsString(duplicates));
+    model.put("nearDuplicateIndicesSequences", mapper.writeValueAsString(nearDuplicates));
+    List<String> warnings = new ArrayList<>();
+    if (!duplicates.isEmpty()) {
+      warnings.add("This pool contains duplicate indices!");
+    } else if (!nearDuplicates.isEmpty()) {
+      warnings.add("This pool contains near-duplicate indices!");
+    }
+    addConsentWarning(pool, model, warnings);
+    model.addAttribute("warnings", warnings);
+  }
+
+  private void addConsentWarning(Pool pool, ModelMap model, List<String> warnings) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "model". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)